library/script/doe_lcgms.py

```diff
@@ -7,6 +7,14 @@
 
 
 class Scriptor:
+    """
+    NOTE: May 6, 2022 The request body for downloading the spreadsheet has not been stable from one year to the next. 
+    To maintein the script, the current setup requires the user to go to the download page and click on the downloading link. 
+    Then open up the webpage in developer mode (with Chrome in this case). Then look at the network tab and payload to find the exact
+    body was generated by the click and sent as a post request to the aspx server. It is probably worth to monitor whether those 
+    arguments would become stable enough that this manual process won't need to be repeated each time for update. Otherwise, alternative 
+    manual download the spreadsheet and upload as local file might be preferred to current method. 
+    """
```

Comment:
I tried to follow these direction starting from [this link](https://infohub.nyced.org/in-our-schools/operations/lcgms) from the doe lcgms template, but I couldn't get it to work

Reply:
okay happy to take a look at how it was done on your side @SashaWeinstein  whenever you want to chat about this.